build_aarch64_wheel.py

```diff
@@ -189,6 +188,8 @@ def update_apt_repo(host: RemoteHost) -> None:
     host.run_cmd("sudo systemctl stop unattended-upgrades.service || true")
     host.run_cmd("while systemctl is-active --quiet apt-daily.service; do sleep 1; done")
     host.run_cmd("while systemctl is-active --quiet unattended-upgrades.service; do sleep 1; done")
+    host.run_cmd("wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | sudo tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null")
+    host.run_cmd("sudo add-apt-repository 'deb https://apt.kitware.com/ubuntu/ focal main'")
```

Comment:
Why this change is necessary? If you want a newer cmake, let's install int from pip/conda

Reply:
yeah, I added this for newer cmake. Let me check conda/pip based install. thanks!